backend/supervisor/jobs/images.go

```diff
@@ -1,6 +1,7 @@
 package jobs
 
 var JobToImage = map[string]string{
+	"facebook-on-status-create":              "",
```

Comment:
You left it blank because it doesnt have to be started as a job ?

Reply:
Exactly, I don't want to create a new map in the back to determine whether we should start a job or not so I added it to the images map on the supervisor. If the value is "" it means that the AREA is valid but it doesn't need a job